pong.py

```diff
@@ -1,108 +1,39 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-import copy
 import pymlgame
 import random
-import time
 
 from ball import Ball
 from misc import Point
 from paddle import Paddle
+from game_states import WaitingState
 
-import drawing
 
 class Game(object):
     """
     The main game class that holds the gameloop.
     """
+
     def __init__(self, mlhost, mlport, screen_width, screen_height):
         """
         Create a screen and define some game specific things.
         """
         pymlgame.init()
-        self.screen = pymlgame.Screen(mlhost, mlport, screen_width, screen_height)
+        self.screen = pymlgame.Screen(
+            mlhost, mlport, screen_width, screen_height)
         self.clock = pymlgame.Clock(15)
         self.gameover = False
         self.players = {}
-        self.state = "WAITING_FOR_PLAYERS"
+        self.state = WaitingState(self)
         self.init_ball()
 
     def init_ball(self):
         ball_position = Point(self.screen.width // 2, self.screen.height // 2)
-        ball_velocity = Point(random.choice([-1, 1]) * 0.5, random.choice([-1, 1]) * 0.2)
+        ball_velocity = Point(random.choice(
+            [-1, 1]) * 0.5, random.choice([-1, 1]) * 0.2)
```

Comment:
We should pull such magic numbers to some central configuration at a later point. Maybe leave a TODO.